src/olympia/addons/tasks.py

```diff
@@ -750,6 +751,66 @@ def migrate_legacy_dictionary_to_webextension(addon):
 
 
 @task
+def migrate_webextensions_to_git_storage(ids, **kw):
+    log.info(
+        'Migrating add-ons to git storage %d-%d [%d].',
+        ids[0], ids[-1], len(ids))
+
+    addons = Addon.unfiltered.filter(id__in=ids)
+
+    for addon in addons:
+        # Filter out versions that are already present in the git
+        # storage.
+        versions = addon.versions.filter(git_hash='').order_by('id')
+
+        for version in versions:
```

Comment:
You don't have a test about skipping versions with a `git_hash` set, nor do you have a test of an add-on with multiple versions (and it would be nice to have, to test the ordering as well)

Reply:
true, I'll add them

Reply:
Add more tests and actually ordering now by `created` from old to new, c3d2947e57